app/lib/features/room/widgets/notifications_settings_tile.dart

```diff
@@ -19,9 +19,15 @@ String? notifToText(String curNotifStatus) {
 
 class _NotificationSettingsTile extends ConsumerWidget {
```

Comment:
https://github.com/acterglobal/a3/assets/51526480/a1a639d5-1588-4b1e-bc08-5c4d1eb90b82

`No-override` setting doesn't looks like setting.

Reply:
gotta check this. thanks for pointing it out!

Reply:
this is an outstanding bug already discussed in https://github.com/acterglobal/a3/issues/1093 . Not sure why this is the case, but I'd like to de-couple it from this PR review as it might be a longer debugging session before we'll find it...

Reply:
Ahh I see, it looks like the same issue. Let's ignore it for now and fixed it on later PR.